pkgs/development/python-modules/fyta-cli/default.nix

```
@@ -4,12 +4,13 @@
   buildPythonPackage,
   fetchFromGitHub,
   hatchling,
+  mashumaro,
   pythonOlder,
 }:
 
 buildPythonPackage rec {
   pname = "fyta-cli";
-  version = "0.5.1";
+  version = "0.6.2";
```

```suggestion
  version = "0.6.3";
```
or 0.6.6 if you want